type-definitions/almin.js.flow

```diff
@@ -63,9 +66,9 @@ declare class UseCaseContext {
 }
 
 declare class UseCaseExecutor {
-    onWillExecuteEachUseCase(handler: (useCase: UseCase, args: mixed) => mixed): void;
-    onDidExecuteEachUseCase(handler: (useCase: UseCase) => mixed): void;
-    onCompleteExecuteEachUseCase(handler: (useCase: UseCase) => mixed): Function;
+    onWillExecuteEachUseCase(handler: (payload: DispatcherPayload, meta: DispatcherPayloadMeta) => mixed): void;
+    onDidExecuteEachUseCase(handler: (payload: DispatcherPayload, meta: DispatcherPayloadMeta) => mixed): void;
+    onCompleteExecuteEachUseCase(handler: (payload: DispatcherPayload, meta: DispatcherPayloadMeta) => mixed): Function;
```

Comment:
Apply unified arguments to `Context#on*` methods.

We will improve this maybe by preparing each Payload class

- `WillExecutePayload`
- `DidExecutePayload`
- `CompleteExecutePayload`